doc/1/controllers/realtime/subscribe/snippets/document-notifications-leave-scope.cs

```diff
@@ -0,0 +1,39 @@
+NotificationHandler listener = (notification) => {
+  string scope = notification.Scope.ToString();
+  if (scope == "out") {
+    Console.WriteLine($"Document moved {scope} from the scope");
+  } else {
+    Console.WriteLine($"Document moved {scope} the scope");
+  }
+};
+
+try {
+  // Subscribes to notifications when document leaves the scope
+  JObject filters = JObject.Parse("{ range: { age: { lte: 20 } } }");
+  SubscribeOptions options = new SubscribeOptions();
+  options.Scope = "out";
```

Comment:
Either:
```suggestion
  options.Scope = "all";
```

Or you have to update this whole example because there won't be any `in` notifications sent.